mage_ai/frontend/tests/base.ts

```diff
@@ -15,8 +28,14 @@ export const test = base.extend<{ page: void; }>({
      */
     await expect(page.getByRole('button', { name: 'New' })).toBeVisible({ timeout: 10000 });
```

Comment:
@edmondwinston Let's move the `visibleTimeout` constant from the `mage_ai/frontend/tests/pipelines.spec.ts` file  to the `base.ts` file instead and import it as needed, since we use the same timeout in a couple different places and can update it in 1 place if needed.

Reply:
Moved, also increased it to 15s.